compiler/hie-core/src/Development/IDE/Core/Shake.hs

```diff
@@ -382,7 +423,9 @@ instance Show k => Show (Q k) where
 
 -- | Invariant: the 'v' must be in normal form (fully evaluated).
 --   Otherwise we keep repeatedly 'rnf'ing values taken from the Shake database
-data A v = A (Value v) (Maybe BS.ByteString)
+-- Note (MK) I am not sure why we need the ShakeValue here, maybe we
```

Comment:
Just to be clear: I am not sure why we had the ByteString here before either so my confusion is not caused by this PR :slightly_smiling_face: